kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/Constants.java

```diff
@@ -142,12 +142,11 @@ private Constants() {
     public static final String SCRAPER_LABEL_KEY = "user-test-app";
     public static final String SCRAPER_LABEL_VALUE = "scraper";
     public static final String SCRAPER_NAME = "Scraper";
-    public static final String CURL_IMAGE = Constants.DOCKER_REGISTRY_GCR_MIRROR + "/curlimages/curl:8.13.0";
+    public static final String CURL_IMAGE = Constants.DOCKER_REGISTRY_GCR_MIRROR + "/curlimages/curl:8.4.0";
```

Comment:
This is the latest version supported in s390x architecture

Reply:
Can we give this an env override purely for `s390x`? The curl image should get bumped by reonvate or dependabot